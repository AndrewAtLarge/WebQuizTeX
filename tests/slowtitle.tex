% should give an error as the title is not in the preamble
\documentclass{webquiz}
\begin{document}

  \title{should be in preamble}

\end{document}
