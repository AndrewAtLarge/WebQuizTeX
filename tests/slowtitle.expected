! Package webquiz Error: \title can only be used in the preamble.
